clients/src/test/java/org/apache/kafka/common/metrics/MetricsTest.java

```diff
@@ -49,7 +49,7 @@
 
     @Before
     public void setup() {
-        this.metrics = new Metrics(config, Arrays.asList((MetricsReporter) new JmxReporter()), time);
+        this.metrics = new Metrics(config, Arrays.asList((MetricsReporter) new JmxReporter()), time, true);
```

Comment:
@auradkar Just a **tip** (no need to change, up to you): you can use ``Collections.singletonList()`` instead of ``Arrays.asList`` when dealing with only a single element collection. ;-)